### PR TITLE
Blockquote margins

### PR DESCRIPTION
**Pull request by tpoisot**
Made a change to avoid the upper margin when using `h2` within a blockquote

blockquote

### CODE REVIEW

**Comment by rgaiacs** [comment]
@tpoisot Thanks for this. I'm -1 to merge this because (1) we have a script to convert some blockquotes to divs and (2) maybe we need to use `h3` instead of `h2`.

Also I'm cleaning up our CSS at #21 and after that will try improve it. If you want to help send me a email and I will reply with more information.

**Comment by aaren** [comment]
@r-gaia-cs this won't affect the blockquotes that become divs - they never see any blockquote specific css. 

([this](https://github.com/swcarpentry/lesson-template/blob/gh-pages/tools/blockquote2div.py) is what we're talking about @tpoisot)

**Comment by tpoisot** [comment]
Actually these should probably not be blockquotes at all, but div with a
class argument of something like "prerequisites". Tell me when you are done
at #21 and I'll have another go.

Le vendredi 14 novembre 2014, Raniere Silva notifications@github.com a
écrit :

> @tpoisot https://github.com/tpoisot Thanks for this. I'm -1 to merge
> this because (1) we have a script to convert some blockquotes to divs and
> (2) maybe we need to use h3 instead of h2.
> 
> Also I'm cleaning up our CSS at #21
> https://github.com/swcarpentry/lesson-template/pull/21 and after that
> will try improve it. If you want to help send me a email and I will reply
> with more information.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/swcarpentry/lesson-template/pull/22#issuecomment-63035195
> .

## 

Sent from my iPhone.

**Comment by aaren** [comment]
@tpoisot good catch. I think the blockquote filter needs to be more permissive.

**Comment by gvwilson** [comment]
@tpoisot can you please shrink the point size for h2 inside blockquote (but leave it as h2), and keep the upper marging diminished as you've done?  With that in place, I'm +1 to merge this.

**Comment by gvwilson** [comment]
What's the status of this one?

**Comment by aaren** [comment]
the blockquote issue was just fixed in #37 so @tpoisot could have another go?

File: css/swc.css [hunk 1]
```diff
@@ -45,7 +45,7 @@ div.chapter h2 {
 /* Block quotations. */
 blockquote {
     margin: 1em;
-    padding: 1em 1em .5em 1em;
+    padding: .7em;
```
Comment (rgaiacs) [inline css/swc.css:48]: Maybe merge only this one?

**Pull request merged**